cmd/csi-snapshotter/main.go

```diff
@@ -135,8 +136,17 @@ func main() {
 	}
 
 	klog.V(2).Infof("CSI driver name: %q", driverName)
-	metricsManager.SetDriverName(driverName)
-	metricsManager.StartMetricsEndpoint(*metricsAddress, *metricsPath)
+
+	// Prepare http endpoint for metrics + leader election healthz
+	mux := http.NewServeMux()
+	cmm := metrics.NewCSIMetricsManagerForSidecar(driverName)
```

Comment:
I believe this is the correct sequence:
https://github.com/kubernetes-csi/external-attacher/pull/278/files

Reply:
Done